src/Settings.hs

```diff
@@ -98,11 +98,24 @@ programPath context@Context {..} = do
         let installPath = if install then inplaceInstallPath package else path
         return $ installPath -/- programName context <.> exe
 
--- TODO: Set this from command line
--- | Stage1Only flag.
-stage1Only :: Bool
-stage1Only = defaultStage1Only
-
 -- | Install's DESTDIR setting.
 getDestDir :: Action FilePath
 getDestDir = fromMaybe "" <$> cmdInstallDestDir
+
+crossCompiling :: Bool
+crossCompiling = False
```

Comment:
We only allow GHC users to edit `UserSettings.hs`, so `crossCompiling` and `stage1Only` flags should be moved there.

Reply:
Emm...sounds good.